eng/source-build-release-announcement.md

```diff
@@ -0,0 +1,5 @@
+[Release Notes]($RELEASE_NOTES_URL) | [Tag]($TAG_URL)
```

Comment:
A general question:  Should this reside in the dotnet/source-build repo?  This is not specific to dotnet/installer.

If we keep this in dotnet/installer, I think we should move the source-build eng infrastructure into a source-build pipelines subfolder.  Eventually the stuff in src/sourcebuild/arcade should be moved here as well.

Reply:
If the announcement template goes in dotnet/source-build, then I guess there's no reason that the entire pipeline couldn't go there as well, since we have the goal of not tying the pipeline to a specific commit of dotnet/installer.

Reply:
My comment wasn't clear, I was asking about the pipeline as well as the template (e.g. this entire PR).  The pipeline feels broader than installers.